pages/account/statistics.js

```diff
@@ -95,15 +96,10 @@ export async function getServerSideProps(context) {
 }
 
 export default function Statistics({ data, profile, progress, BASE_URL }) {
-  const dateTimeStyle = {
-    dateStyle: "short",
-  };
   const dailyViews = data.profile.daily.slice(-30).map((day) => {
     return {
       views: day.views,
-      date: new Intl.DateTimeFormat("en-GB", dateTimeStyle).format(
-        new Date(day.date)
-      ),
+      date: dateFormat({ format: "short", date: day.date }),
```

Comment:
This code will have hydration issues.

When SSR is run, the locale will be "en-GB" (using Vercel in London).
On the client, it will be local (ex "en-US"), which if different, will force a complete re-render.

The way to avoid this, is to do an initial conversion in getServerSideProps. Store the value in a state variable. Then update on render with a useEffect.

Reply:
Oh wow ok, I didn't realise that, so that problem exists already?

Reply:
No.

It is caused by the switch from "en-GB" to local.

Reply:
Oh I see! Thank you!

I tried to do the dynamic way with useEffect, but had some weird issues. I will put it back to `en-GB` and further improvements can come later

Reply:
Pushed, hopefully this is ok now?

Reply:
Looks good for this PR. I will look into making some improvements in a future PR.

Reply:
Great! Thank you Dan ❤️